py3status/modules/bluetooth.py

```diff
@@ -55,47 +52,28 @@ class Py3status:
     # available configuration parameters
     cache_timeout = 10
     device_separator = '|'
-    format = '{name}'
-    format_no_conn = 'OFF'
-    format_no_conn_prefix = 'BT: '
-    format_prefix = 'BT: '
+    format = 'BT[: {format_device}]'
```

Comment:
I'm sure I took care of `format_prefix` just now. How should I check for `{name}` or `{mac}` in format but ignore if `format_device` exists? Meh. I can test it. Could `format_no_conn` thing be done w/ the formatter?... Something like...
```
'\?if=!format_device OFF|{}'{{format_device}}'.format(config['format_prefix'])
```
I dunno what I'm doing here. Too messy for me.